php/libraries/Utility.class.inc

```diff
@@ -216,12 +219,12 @@ class Utility
     /**
      * Returns a list of study Subprojects
      *
-     * @param integer $projectID The project for which you would like
+     * @param integer|null $projectID The project for which you would like
      *                           to get the subprojects
      *
      * @return array       an associative array("SubprojectID" => "Subproject title")
      */
-    static function getSubprojectList($projectID = null)
+    static function getSubprojectList(int $projectID = null): array
```

Comment:
I know Travis is happy but should't it be `?int $projectID = null` (same for line 257)?